homeassistant/components/airscape/fan.py

```diff
@@ -0,0 +1,173 @@
+"""Platform for Airscape fan integration."""
+import logging
+import airscape
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+
+# Import the device class from the component that you want to support
+from homeassistant.components.fan import (
+    SUPPORT_SET_SPEED,
+    PLATFORM_SCHEMA,
+    FanEntity,
+    DOMAIN,
+)
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_TIMEOUT, CONF_MINIMUM
+
+DEFAULT_TIMEOUT = 5
+DEFAULT_MINIMUM = 1
+
+_LOGGER = logging.getLogger(__name__)
+
+# Validation of the user's configuration
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_NAME): cv.string,
+        vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
+        vol.Optional(CONF_MINIMUM, default=DEFAULT_MINIMUM): cv.positive_int,
+    }
+)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Airscape Fan platform."""
+    # Setup connection to the fan
+    host = config.get(CONF_HOST)
+    name = config.get(CONF_NAME)
+    minimum = config.get(CONF_MINIMUM)
+    timeout = config.get(CONF_TIMEOUT)
+
+    try:
+        device = airscape.Fan(host, timeout)
+    except (airscape.exceptions.Timeout, airscape.exceptions.ConnectionError):
+        _LOGGER.error(
+            "Cannot connect to %s.  " "Device did not respond to API request", host
```

Comment:
Please remove the split in the string. It's a side effect after black formatting.